src/main/resources/i18n/Messages_en.properties

```diff
@@ -242,6 +242,19 @@ notification.seedBank.unknownAutomationTriggered.app.title={0} triggered at {1}
 notification.seedBank.unknownAutomationTriggered.email.body.1=Automation {0} has been triggered at {1}.
 notification.seedBank.unknownAutomationTriggered.email.body.2=Additional details\: {0}
 notification.seedBank.unknownAutomationTriggered.email.subject={0} triggered at {1}
+notification.support.requestSubmitted.email.body.1=Thank you for contacting Terraware support.
+notification.support.requestSubmitted.email.body.2=We rely on our users to help make Terraware better and appreciate hearing from you.
+notification.support.requestSubmitted.email.body.3=Within the next few days, we may reach out to obtain additional information that would help us investigate the issue or to provide you with the status of a bug fix.
+notification.support.requestSubmitted.email.body.4=In the meantime, you may find helpful information in the [Terraware - Knowledge Base].
+notification.support.requestSubmitted.email.body.5=For your reference, here are the details of your request\:
+# {0} is the issue key
+notification.support.requestSubmitted.email.body.6=Request Number\: {0}
+# {0} is the summary of the request
+notification.support.requestSubmitted.email.body.7=Summary\: {0}
+# {0} is the description of the request
+notification.support.requestSubmitted.email.body.8=Description\: {0}
+# {0} is the name of the request type, {1} is the key of the request
+notification.support.requestSubmitted.email.subject={0} Received: {1}
```

Comment:
This may be the best we can do for now, but I think long term we will probably want a separate string in this file for each request type since in some languages (Arabic, for example) the verb in this phrase would need to change depending on the noun.

Also, am I correct in assuming that at the moment, the request type will always be in English here even if the user's language is set to Spanish, since the name is an opaque string we get back from Atlassian?

Reply:
Yes that name is always in English for now. I think it gets tricky since we will need to translate on the fly.

Reply:
Maybe down the line, once the Request types are more finalized, we can take the reverse approach, and have Terraware server actually create the request types in Jira via API if they don't already exist. 

At that point, we can define all the service request types we want to support in a table, and translate any enum names, etc. 

Right now it is hard to translate everything from Jira because we are treating that as the ground truth.